### DISCUSSION TITLE
Does DragronFly Cloud supports high availability and persistence for non-cache data?

### DISCUSSION THREAD

**Discussion by webaddicto** [dragonflydb/dragonfly#3832] [General]
I would like to use DragonFly DB managed cloud for a project that needs high availability and persistence, so data stored there is not used as cache but as database and is considered important. Main focus is that data is not lost in any case if possible, and we have high availability and failover support so we can be sure in case of an issue we don't lose the data (at max we have data not older than 1 second ago) and the instance is always up.

Is this supported on DragonFly cloud?

Thanks!

**Comment by ashotland** [comment]
Hi @webaddicto,

Dragonfly Cloud support high availability, with primary and replica and automatic failover.

It doesn't support persistence. Hourly backups can be scheduled. 

For more information please see https://www.dragonflydb.io/docs/cloud
